src/System.IO.Pipes/src/System/IO/Pipes/PipeOptions.cs

```diff
@@ -10,5 +10,6 @@ public enum PipeOptions
         None = 0x0,
         WriteThrough = unchecked((int)0x80000000),
         Asynchronous = unchecked((int)0x40000000),  // corresponds to FILE_FLAG_OVERLAPPED
+        CurrentUserOnly = unchecked((int)0x20000000)
```

Comment:
What happens if Windows adds a new option to the CreateNamedPipe flags with this number? I guess that's OK, it's only our convention that the numbers currently match what we send to Windows.

Reply:
This was the proposed value in the original API proposal, from @stephentoub: 

>  but for extra good measure we should also choose a value that's not currently accepted by CreateNamedPipe